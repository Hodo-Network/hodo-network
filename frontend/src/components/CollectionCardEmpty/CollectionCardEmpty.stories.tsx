import React from "react";
import { Story, Meta } from "@storybook/react";

import { CollectionCardEmpty } from "./CollectionCardEmpty";

export default {
  title: "Components/CollectionCardEmpty",
  component: CollectionCardEmpty,
  decorators: [
    (Story) => (
      <div className='w-72'>
        <Story />
      </div>
    ),
  ],
} as Meta;

const Template: Story = (args) => <CollectionCardEmpty {...args} />;

export const Default = Template.bind({});
Default.args = {};
