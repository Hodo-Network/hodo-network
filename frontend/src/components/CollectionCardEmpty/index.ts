import { CollectionCardEmpty } from './CollectionCardEmpty';
export { CollectionCardEmpty };
