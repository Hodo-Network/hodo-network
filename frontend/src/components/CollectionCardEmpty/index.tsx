import { CollectionCardEmpty } from "./CollectionCardEmpty";
export default CollectionCardEmpty;
