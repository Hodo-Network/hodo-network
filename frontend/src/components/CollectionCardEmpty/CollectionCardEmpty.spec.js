import ReactDOM from "react-dom";
import { cleanup } from "@testing-library/react";
import renderer from "react-test-renderer";
import CollectionCardEmpty from "./index";

afterEach(cleanup);

test("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<CollectionCardEmpty />, div);
});

test("matches snapshot", () => {
  const tree = renderer.create(<CollectionCardEmpty />).toJSON();
  expect(tree).toMatchSnapshot();
});
