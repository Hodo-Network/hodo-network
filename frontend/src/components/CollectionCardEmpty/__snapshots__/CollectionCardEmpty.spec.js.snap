// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="flex flex-col rounded-md overflow-hidden transition-transform duration-300 transform hover:-translate-y-1 border-dashed border border-gray-400 dark:border-gray-700"
>
  <div
    className="flex-shrink-0 w-full"
  >
    <div
      className="h-60"
    />
  </div>
  <div
    className="flex-1 bg-white dark:bg-gray-800 p-4 flex flex-col justify-between"
  >
    <div
      className="flex-1"
    >
      <p
        className="text-base font-semibold text-gray-900 dark:text-gray-100 capitalize"
      >
        Nothing to display
      </p>
    </div>
  </div>
</div>
`;
